### ISSUE TITLE
Parallel coverage fails in CI and blocks merging

### ISSUE THREAD

**Issue by Kouprin** [nearprotocol/nearcore#1861]
May be related to https://github.com/nearprotocol/nearcore/pull/1823.
```
149 ========= error: kcov /home/gitlab-runner/builds/V2EEF62L/0/near-protocol/nearcore/target/debug/fork_choice-95339fe0c5169227 fail, exit code 101 cause coverage fail
150 ========= kcov /home/gitlab-runner/builds/V2EEF62L/0/near-protocol/nearcore/target/debug/announce_account-6681bd4cbb3c537f done
151 ========= kcov /home/gitlab-runner/builds/V2EEF62L/0/near-protocol/nearcore/target/debug/test_rs_contract-eb9ff7b6eb45e8cd done
152 ========= kcov /home/gitlab-runner/builds/V2EEF62L/0/near-protocol/nearcore/target/debug/chunks_management-2c2c0b5f4f08b43d done
153 ========= kcov /home/gitlab-runner/builds/V2EEF62L/0/near-protocol/nearcore/target/debug/test_registers-072aa40141e62983 done
154 ========= kcov /home/gitlab-runner/builds/V2EEF62L/0/near-protocol/nearcore/target/debug/keypair_generator-cb7b968a8369b866 done
155 ========= kcov /home/gitlab-runner/builds/V2EEF62L/0/near-protocol/nearcore/target/debug/near_epoch_manager-9769ea25decd914b done
156 ========= kcov /home/gitlab-runner/builds/V2EEF62L/0/near-protocol/nearcore/target/debug/helpers-d6cebc06003291b6 done
157 ========= kcov /home/gitlab-runner/builds/V2EEF62L/0/near-protocol/nearcore/target/debug/peer_handshake-ac5b616682d4d386 done
158 ========= kcov /home/gitlab-runner/builds/V2EEF62L/0/near-protocol/nearcore/target/debug/rpc_query-a05a3311485bb0cd done
159 ========= kcov /home/gitlab-runner/builds/V2EEF62L/0/near-protocol/nearcore/target/debug/fixtures-eff7b8ce95233dac done
160 ========= kcov /home/gitlab-runner/builds/V2EEF62L/0/near-protocol/nearcore/target/debug/test_evil_contracts-3b5618554f932d74 done
161 ========= run in 2 workers
162 ========= coverage merged to /home/gitlab-runner/builds/V2EEF62L/0/near-protocol/nearcore/scripts/../target/cov6/71
163 Traceback (most recent call last):
164   File "scripts/parallel_coverage.py", line 109, in <module>
165     subprocess.check_output(['mv', merged_coverage, f'{current_path}/../target/merged_coverage'])
166   File "/usr/lib/python3.6/subprocess.py", line 356, in check_output
167     **kwargs).stdout
168   File "/usr/lib/python3.6/subprocess.py", line 438, in run
169     output=stdout, stderr=stderr)
170 subprocess.CalledProcessError: Command '['mv', '/home/gitlab-runner/builds/V2EEF62L/0/near-protocol/nearcore/scripts/../target/cov6/71', '/home/gitlab-runner/builds/V2EEF62L/0/near-protocol/nearcore/scripts/../target/merged_coverage']' returned non-zero exit status 1.
174 ERROR: Job failed: exit status 1
```
Example https://gitlab.com/near-protocol/nearcore/-/jobs/378366962.

**Comment by Kouprin** [comment]
https://gitlab.com/near-protocol/nearcore/-/jobs/377771049
same for another PR

**Comment by ailisp** [comment]
It''s fixed in #1856
